test/functional/conftest.py

```diff
@@ -31,6 +31,8 @@ def pytest_addoption(parser):
     parser.addoption('--instance-type')
     parser.addoption('--accelerator-type', default=None)
     parser.addoption('--region', default='us-west-2')
+    parser.addoption('--framework-version', default='1.12.0')
```

Comment:
can we read this from the python sdk?

Reply:
fair enough. this is probably something that should be done with the other integ test suites as well.